### ISSUE TITLE
[Voice Access] Expander expand/collapse button cannot be targeted

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/microsoft-ui-xaml#7817]
### Describe the bug

Voice access has a feature where each interactable control can have a number associated with it for targeting. Expanders do not have a number tag present on the expand/collapse button.

An important thing to note is that if a button is embedded as the header content of an expander, that button does get a number tag.

### Steps to reproduce the bug

1. Open WinUI 2 Gallery > Expander
2. Open Voice Access
3. Say "Show numbers"

### Expected behavior

A number tag should appear on each expander.

### Screenshots

Unfortunately, summoning snipping tool removes the number tags. However, the accessibility team managed to get a screen capture of the issue in Windows Terminal.

[Actual.and.Note.Issue.zip](https://github.com/microsoft/microsoft-ui-xaml/files/9760378/Actual.and.Note.Issue.zip)

Though Windows Terminal is a XAML Islands app, this issue repros with the WinUI 2 Gallery (a non XAML Islands app).

### Windows app type

- [ ] UWP
- [ ] Win32

### Device form factor

Desktop

### Windows version

Windows 11 (22H2): Build 22621

**Comment by kmahone** [comment]
I believe that this is fixed in WinAppSDK 1.4. Can you try the latest preview release?
